src/hooks/useCheckTypeGroupOptions.tsx

```diff
@@ -80,7 +79,6 @@ export const CHECK_TYPE_GROUP_OPTIONS: CheckTypeGroupOption[] = [
     description: `Monitor the availability and performance of a website using a real browser.`,
     value: CheckTypeGroup.Browser,
     icon: `globe`,
-    status: CheckStatus.PRIVATE_PREVIEW,
```

Comment:
Similar problem to above where I wanted all of the associated information colocated so it is easier to reason with.